documentation/source/users/arkane/input.rst

```diff
@@ -1037,6 +1037,29 @@ for all network reactions (both directions if reversible) at all requested condi
 automatically generated per network reaction, showing the semi-normalized sensitivity coefficients at all conditions.
 
 
+Bond Additivity Correction Fitting
+==================================
+
+If calculated data is available in the reference database, Arkane can fit and save bond additivity corrections (BACs).
+There are two different types available: Petersson-type, which fits one parameter for each bond type, and Melius-type,
+which fits three parameters per atom type and an optional molecular parameter. Each BAC fitting calculation must be
+specified using a ``bac()`` function, which accepts the following parameters:
```

Comment:
Should we cite literature again here for Melius (perhaps Petersson as well, though that's what most people expect)?